models/user/user.go

```diff
@@ -355,6 +353,44 @@ func GetUserFollowing(ctx context.Context, u, viewer *User, listOptions db.ListO
 	return users, count, err
 }
 
+// GetUserFollowersCount returns count of user's followers.
+func GetUserFollowersCount(ctx context.Context, u, viewer *User, listOptions db.ListOptions) (int64, error) {
+	sess := db.GetEngine(ctx).
```

Comment:
Maybe all of this should be extracted as a function, so it could be shared with `GetUserFollowers`.